src/framework/MOM_diag_mediator.F90

```diff
@@ -2384,6 +2385,21 @@ integer function register_diag_field(module_name, field_name, axes_in, init_time
     enddo ! i
   enddo
 
+  dimensions = ""
+  if (axes_in%is_h_point)   dimensions = trim(dimensions)//" ,yh, xh"
```

Comment:
Overall I like what is done here.  I have two questions, though, to consider in perhaps minorly revising this code:
1. Why is there a space before the comma on lines 2389-2392 and similarly on lines 3133-3136 and not after?  (I.e., why is the end of this lines `" ,yh, xh"` instead of `", yh, xh"`?
2. Why is the order of dimensions here given as (for example) "yh, xh, zl" instead of the "xh, yh, zl" that might seem more natural?